src/app/offers/offer-details/offer-details.component.ts

```diff
@@ -25,7 +25,9 @@ export class OfferDetailsComponent implements OnInit {
     isOverview = false;
     isPortfolio = false;
     userConfirmed = true;
-    currentLocation = encodeURIComponent(window.location.href);
+    projectID = this.route.snapshot.params.id;
+    projectOverviewLink = window.location.host + '/overview/' + this.projectID + '/discover';
+    currentLocation = encodeURIComponent(this.projectOverviewLink);
```

Comment:
Don't refer to other properties in the declaration - this should be done in `constructor`